nfs.tf

```diff
@@ -1,10 +1,15 @@
 resource "openstack_compute_instance_v2" "nfs-server" {
+  depends_on = ["openstack_networking_subnet_v2.internal"]
```

Comment:
is this necessary, with the `network` block below? my understanding was it would wait properly?

Reply:
Probably not. I can easily run a test without it today.

Reply:
Anyway, looks good from my end, glad this worked so easily!

Reply:
Not needed.